File: 551-StudentAttendanceRecordI/551-StudentAttendanceRecordI.java
// Last updated: 10/08/2026, 14:56:56
1class Solution {
2    public int[] sortedSquares(int[] arr) {
3        for (int i=0;i<arr.length;i++)
4        {
5            arr[i]=arr[i]*arr[i];
6        }
7        Arrays.sort(arr);
8        return arr;
9    }
10}

File: 551-StudentAttendanceRecordI/551-StudentAttendanceRecordI.py
# Last updated: 10/08/2026, 14:50:38
1class Solution(object):
2    def heightChecker(self, h):
3        arr=[]
4        for i in h:
5            arr.append(i)
6        arr.sort()
7        c=0
8        for i in range(len(h)):
9            if h[i] != arr[i]:
10                c+=1 
11        return c
12        